prebid-server/developers/add-a-module-go.md

```diff
@@ -0,0 +1,266 @@
+---
+layout: page_v2
+sidebarType: 5
+title: Prebid Server | Developers | Adding a Go Module
+
+---
+
+# Prebid Server - Adding a Go Module
+{: .no_toc}
+
+* TOC
+{:toc }
+
+## Overview
+
+This document details how to make a module for PBS-Go.
+
+You will want to be familiar with the following background information:
+
+- the [module overview](/prebid-server/developers/add-a-module.html) 
+- the [PBS-Go Modularity Tech Spec](https://docs.google.com/document/d/1CmamniQpwcI3p0_rHe2F17zV4sEhzpOdrqU7zuZVZ_I/edit?usp=sharing)
+
+### Contributing
+
+Check out the [PBS-Go contribution guide](https://github.com/prebid/prebid-server/blob/master/docs/developers/contributing.md) before introducing any code changes.
+
+## Module Directory Layout
+
+The Prebid Server repository contains a package `modules` located in the root project directory. It includes all available PBS modules. So, in order to add a new module, fork the repository and create a folder with the desired name inside the `modules` folder with the following structure:
+
+```
++- prebid-server/
+  +- modules/                <- package with modules that implement various hooks
+    +- builder.go            <- contains a list of all available modules
+    +- {YOUR_VENDOR_NAME}/   <- top-level package used to group modules from the same vendor
+      +- {YOUR_MODULE_NAME}/ <- package with source code of your module
```

Comment:
Could you please specify name format for vendor and module directory and what symbols are not aloowed?

Reply:
Of course, added note about valid module directory name.